### DISCUSSION TITLE
Unable to add blogs - "Error 1: The device is not ready"

### DISCUSSION THREAD

**Discussion by Tygerboi** [TumblThreeApp/TumblThree#542] [Q&A (Question & Answer)]
For the last couple of months I have been unable to download any Tumblr blogs using TumblThree.  When I enter the blog URL (which works in Edge - latest build) into Edge the pages load just fine, as they also do in Chrome.  I am running the latest version of TumblThree - 2.1.3.0442 - on Windows 10 Pro v22H2.  There is plenty of hard drive space available.  I have had the same authentication issues as in the post link below, but I was still somehow able to download blogs.  The "device not ready" error started happening since the previous Tumblr update to the one above.

I have tried completely uninstalling TumblThree - deleting all app folders in my user profile folders, AppData, etc - I searched my entire C: drive for "Tumblr" and "TumblThree" (with "show hidden files" enabled) and deleted everything.  I downloaded the TumblThree setup from Github again and then installed to a different folder from before, but all of this made no difference whatsoever.  I have shut down my AV/VPN apps and tried again, but the same problems still occur.  I have changed literally nothing else on my PC since the time that TumblThree worked, and have not even changed the blog URLs that I'm trying to enter into TumblThree.  Examples of the URLs I'm using are:
https //myblog.tumblr.com/  - this resolves to https://www.tumblr.com/myblogname    in Edge, but TumblThree has never let me enter the URL in that format.
https://www.tumblr.com/liked/by/myblogname

I am now out of ideas.  Please advise.

https://github.com/TumblThreeApp/TumblThree/discussions/451

**Comment by thomas694** [comment]
I don't see any changes from 2.12.0 to 2.13.0.442 that should cause such an error. Next time, can you hover over the error in the error bar or also search for the error in the log file (%LOCALAPPDATA%\TumblThree)? Maybe it tells us more.
Are you using `Collections` or have tried it out before? One of the download paths there or a download location of a blog points to a location that's currently unavailable, I guess.

General information:
[wiki/How-to-Uninstall-the-Application?](https://github.com/TumblThreeApp/TumblThree/wiki/How-to-Uninstall-the-Application%EF%BC%9F)
Both URL formats refer to the same thing. For adding new blogs the URLs can be entered in the new and old format. However, TumblThree continues to store them internally in the old format.

**Reply by evildisko** [reply]
I'm having the same problem. When I hover over the error it says "Internal Error description the device is not ready." When I updated to the current version I lost all of my blogs too. I tried uninstalling and reinstalling with no effect.

**Reply by thomas694** [reply]
The next time the error occurs, please check the log file (%LOCALAPPDATA%\TumblThree) and give us the complete error message.

As long as you haven't deleted your original download location, all you need to do is go back to Settings and re-enter your download location. Your blogs will be back either immediately or after restarting the app.
